lib/daru.rb

```diff
@@ -65,11 +65,18 @@ def create_has_library(library)
   create_has_library :nyaplot
 end
 
-autoload :Spreadsheet, 'spreadsheet'
-autoload :CSV, 'csv'
+[['reportbuilder', '~>1.4'], ['spreadsheet', '~>1.1.1']].each do |lib|
+  begin
+    gem lib[0], lib[1]
+    require lib[0]
+  rescue LoadError
+    STDERR.puts "\nInstall the #{lib[0]} gem version #{lib[1]} for using"\
```

Comment:
I will try to come up with a PR then :)
Something like `Daru::Extensions::Reporting`?